presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllAnalyzableTypesTableDefinitions.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.tests.hive;
+
+import com.teradata.tempto.fulfillment.table.TableDefinitionsRepository;
+import com.teradata.tempto.fulfillment.table.hive.HiveDataSource;
+import com.teradata.tempto.fulfillment.table.hive.HiveTableDefinition;
+
+import java.util.Optional;
+import java.util.concurrent.ThreadLocalRandom;
+
+import static com.teradata.tempto.fulfillment.table.hive.InlineDataSource.createResourceDataSource;
+
+public final class AllAnalyzableTypesTableDefinitions
+{
+    private AllAnalyzableTypesTableDefinitions()
+    {
+    }
+
+    @TableDefinitionsRepository.RepositoryTableDefinition
+    public static final HiveTableDefinition ALL_ANALYZABLE_HIVE_TYPES_TEXTFILE = tableDefinitionBuilder("TEXTFILE", Optional.of("DELIMITED FIELDS TERMINATED BY '|'"))
+            .setDataSource(getTextFileDataSource())
+            .build();
+
+    private static HiveTableDefinition.HiveTableDefinitionBuilder tableDefinitionBuilder(String fileFormat, Optional<String> rowFormat)
+    {
+        return HiveTableDefinition.builder("all_analyzable_types")
+                .setCreateTableDDLTemplate("" +
+                        "CREATE %EXTERNAL% TABLE %NAME%(" +
+                        "   c_tinyint            TINYINT," +
+                        "   c_smallint           SMALLINT," +
+                        "   c_int                INT," +
+                        "   c_bigint             BIGINT," +
+                        "   c_float              FLOAT," +
+                        "   c_double             DOUBLE," +
+                        "   c_decimal            DECIMAL," +
+                        "   c_decimal_w_params   DECIMAL(10,5)," +
+                        "   c_timestamp          TIMESTAMP," +
+//                        "   c_date               DATE," +
```

Comment:
It should be just removed.
The reason why don't we want date column here is because product tests are run on cdh5 which has Hive 1.1.
And that version does not support ANALYZE on tables with date columns.
It fails with: 
` Error while compiling statement: FAILED: UDFArgumentTypeException Only integer/long/timestamp/float/double/string/binary/boolean/decimal type argument is accepted but date is passed.`

We should alter the test as soon as we move to Hive 1.2. Actually the test passes fine with date column if run on HDP distribution with Hive 1.2. 

I will file an issue to migrate main product tests run to HDP.